lldb/source/DataFormatters/FormatManager.cpp

```diff
@@ -731,7 +731,7 @@ FormatManager::FormatManager()
   EnableCategory(m_vectortypes_category_name, TypeCategoryMap::Last,
                  lldb::eLanguageTypeObjC_plus_plus);
   EnableCategory(m_runtime_synths_category_name, TypeCategoryMap::Last,
-                 {lldb::eLanguageTypeObjC_plus_plus, lldb::eLanguageTypeSwift});
+                 lldb::eLanguageTypeSwift);
```

Comment:
Snap. Are you sure this is correct?

Reply:
No, that's why I asked for review :)

Reply:
I think this is wrong, but to be sure, try the same patch on a stable branch and see if it passes. If not, then we probably need to reintroduce the API that takes a vector upstream.

Reply:
Seems like calling it twice (one for ObjC and once for Swift) is identical to the vector version. That seems less invasive.